test/TypeCoercion/integer_literals.swift

```diff
@@ -63,6 +63,6 @@ func chaining() {
 func memberaccess() {
   Int32(5._value) // expected-warning{{unused}}
   // This diagnostic is actually better than it looks, because the inner type is Builtin.Int32, not actually Int32.
-  let x : Int32 = 7._value // expected-error{{cannot convert value of type 'Int32' to specified type 'Int32'}}
+  let x : Int32 = 7._value // expected-error{{cannot convert value of type 'Builtin.Int32' to specified type 'Int32'}}
```

Comment:
I don’t think we should be leaking that (`Builtin`) into diagnostics.

Reply:
No, I think it's fine. It's hard to actually construct an expression of Builtin type, but if you do, the compiler should let you know what's going on.